autoray/lazy/core.py

```diff
@@ -1195,6 +1220,11 @@ def binary_func(x1, x2):
 floordivide = make_binary_func("floordivide", operator.floordiv)
 truedivide = make_binary_func("truedivide", operator.truediv)
 pow_ = make_binary_func("pow", operator.pow)
+gt = make_binary_func('gt', operator.gt)
+ne = make_binary_func('gt', operator.ne)
+lt = make_binary_func('gt', operator.lt)
+ge = make_binary_func('gt', operator.ge)
+le = make_binary_func('gt', operator.le)
```

Comment:
Should these each have their own name?

Reply:
Yes, I forgot to change the name. Fixed it.